src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/Process_StreamTests.cs

```diff
@@ -9,105 +9,112 @@ namespace System.Diagnostics.ProcessTests
 {
     public partial class ProcessTest
     {
+
+        Process CreateProcessError()
+        {
+            Process p = new Process();
+            p.StartInfo.FileName = _processName;
+            p.StartInfo.Arguments = "error";
+            return p;
+        }
+
+        Process CreateProcessInput()
+        {
+            Process p = new Process();
+            p.StartInfo.FileName = _processName;
+            p.StartInfo.Arguments = "input";
+            return p;
+        }
+
+        Process CreateProcessStream()
+        {
+            Process p = new Process();
+            p.StartInfo.FileName = _processName;
+            p.StartInfo.Arguments = "stream";
+            return p;
+        }
+
         [Fact]
-        public static void Process_SyncErrorStream()
+        public void Process_SyncErrorStream()
         {
             Process p = CreateProcessError();
             p.StartInfo.RedirectStandardError = true;
             p.Start();
-            if (!p.HasExited)
-            {
-                string s = p.StandardError.ReadToEnd();
-                p.WaitForExit();
-
-                s = s.Replace("\r", "").Replace("\n", "");
-
-                Assert.Equal("Unhandled Exception: Intentional Exception thrown", s);
-            }
+            Assert.Equal(p.StandardError.ReadToEnd(), "ProcessTest_ConsoleApp.exe error stream\r\n");
+            p.WaitForExit();
         }
 
-        private static StringBuilder s_process_AsyncErrorStream_sb = new StringBuilder();
+        private StringBuilder process_AsyncErrorStream_sb = new StringBuilder();
 
         [Fact]
-        public static void Process_AsyncErrorStream()
+        public void Process_AsyncErrorStream()
         {
-            s_process_AsyncErrorStream_sb.Clear();
+            process_AsyncErrorStream_sb.Clear();
```

Comment:
Same comment as earlier about using a local captured by a lambda rather than using an instance field, e.g.
```
Process p = CreateProcessError();
p.StartInfo.RedirectStandardError = true;
StringBuilder sb = new StringBuilder();
p.ErrorDataReceived += (s,e) => sb.Append(e.Data);
p.Start();
p.BeginErrorReadLine();
p.WaitForExit();
Assert.Equal("...", sb.ToString());
```